pkgs/tools/package-management/poetry-plugin-up/default.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, fetchFromGitHub
+, python3
+}:
+
+let
+  python = python3;
+in
+python.pkgs.buildPythonPackage rec {
+  pname = "poetry-plugin-up";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "MousaZeidBaker";
+    repo = pname;
+    rev = "refs/tags/${version}";
+    hash = "sha256-16p0emvgWa56Km8U5HualCSStbulqyINbC3Jez9Y1n0=";
+  };
+
+  format = "pyproject";
+  nativeBuildInputs = with python.pkgs; [ poetry-core pythonRelaxDepsHook ];
```

```suggestion
  nativeBuildInputs = with python.pkgs; [
    poetry-core
    pythonRelaxDepsHook
  ];

```

one dependency per line and a free line between the lists.